docs/dafni_cli.md

```diff
@@ -2,24 +2,43 @@
 
 ## Installation
 
+The current method of distribution is via an executable program that can be run on the command line. This will take the form of a executable file called `dafni.exe`.
+Due to this, there is no installation at this stage from a user perspective, just a necessity to have a copy of the executable on the computer it is to be used on.
+___
+
 ## Commands
 
-The syntax for these will change as per the requirements i.e.\
+The syntax for these will change as per the requirements i.e.  
 `dafni get model` instead of `dafni model get`.
```

Comment:
Do you mean that the ".exe" will eventually be dropped?

Reply:
So that was already there actually, although reading through it can likely be dropped entirely

Reply:
Oh, that's probably from me then. Yeah, we can drop this.

Reply:
removed